src/PythonClient/src/quixstreams/models/timeseriesbuffer.py

```diff
@@ -202,16 +232,27 @@ def callback(ts: ctypes.c_void_p) -> bool:
     @property
     def custom_trigger(self) -> Callable[[TimeseriesData], bool]:
         """
-            Gets the custom function which is invoked after adding a new timestamp to the buffer. If returns true, TimeseriesBuffer.on_data_received is invoked with the entire buffer content
-            Defaults to none (disabled).
-        """
+        Gets the custom trigger function, which is invoked after adding a new timestamp to the buffer.
+
+        If the custom trigger function returns True, the `TimeseriesBuffer.on_data_received`
+        is invoked with the entire buffer content. By default, this feature is disabled (None).
+
+        Returns:
+            Callable[[TimeseriesData], bool]: Custom trigger function.
+    """
         return self._custom_trigger
 
     @custom_trigger.setter
     def custom_trigger(self, value: Callable[[TimeseriesData], bool]):
         """
-            Sets the custom function which is invoked after adding a new timestamp to the buffer. If returns true, TimeseriesBuffer.on_data_received is invoked with the entire buffer content
-            Defaults to none (disabled).
+        Sets the custom trigger function, which is invoked after adding a new timestamp to the buffer.
+
+        If the custom trigger function returns True, the `TimeseriesBuffer.on_data_received`
```

Comment:
on_data_released not on_data_received. this is a leftover after renaming.
Also, in fact on_dataframe_released and on_raw_released also get raised, depending on what user subscribed to. Saw this issue in other comments too, probably were written before the other two were available.

Possibly need to rephrase these in a way that is more abstracted like, in this scenario
"if the custom trigger function returns True, the buffer releases content and triggers relevant callbacks."